### ISSUE TITLE
Eclipse Dependency issue

### ISSUE THREAD

**Issue by JanoValaska** [Yakindu/solidity-ide#128]
I installed fresh Eclipse Oxygen 3 and when I try to install Yakindu Solidity IDE I get following error:

```
Cannot complete the install because of a conflicting dependency.
  Software being installed: Solidity Editor Feature 0.7.2.201804271436 (com.yakindu.solidity.feature.feature.group 0.7.2.201804271436)
  Software currently installed: Eclipse Java Development Tools 3.13.4.v20180330-0919 (org.eclipse.jdt.feature.group 3.13.4.v20180330-0919)
  Only one of the following can be installed at once: 
    Java Development Tools Core 3.13.50.v20171007-0855 (org.eclipse.jdt.core 3.13.50.v20171007-0855)
    Java Development Tools Core 3.13.100.v20171123-1049 (org.eclipse.jdt.core 3.13.100.v20171123-1049)
    Object Teams Development Tooling Core 3.13.101.v_OTDT_r262_201803031341 (org.eclipse.jdt.core 3.13.101.v_OTDT_r262_201803031341)
    Object Teams Development Tooling Core 3.13.100.v_OTDT_r261_201712031505 (org.eclipse.jdt.core 3.13.100.v_OTDT_r261_201712031505)
    Java Development Tools Core 3.13.101.v20180215-0725 (org.eclipse.jdt.core 3.13.101.v20180215-0725)
    Java Development Tools Core 3.13.102.v20180330-0919 (org.eclipse.jdt.core 3.13.102.v20180330-0919)
    Java Development Tools Core 3.13.0.v20170516-1929 (org.eclipse.jdt.core 3.13.0.v20170516-1929)
    Object Teams Development Tooling Core 3.13.102.v_OTDT_r262_201804012017 (org.eclipse.jdt.core 3.13.102.v_OTDT_r262_201804012017)
  Cannot satisfy dependency:
    From: Solidity Editor Feature 0.7.2.201804271436 (com.yakindu.solidity.feature.feature.group 0.7.2.201804271436)
    To: org.eclipse.jdt.core [3.13.100.v20171123-1049]
  Cannot satisfy dependency:
    From: Eclipse Java Development Tools 3.13.4.v20180330-0919 (org.eclipse.jdt.feature.group 3.13.4.v20180330-0919)
    To: org.eclipse.jdt.core [3.13.102.v20180330-0919]
```

Can anyone help, please?

NOTE: I tried to install Solidity IDE on Eclipse Photon and Neon as well, but I am getting similar dependency issues.

**Comment by kthoms** [comment]
This is a conflict with multiple versions of JDT. Only one JDT version can be installed, the mentioned bundles are singletons. Which Oxygen distro did you use and how did you install it? Did you download a package or used the Eclipse Installer? Please give steps for reproduction.

**Comment by flantony** [comment]
We have just received similar report, that might relate to this issue:
I'm having troubles downloading the Yakindu Solididty Tools on Eclipse Java EE IDE for Web Developers.

Version: Photon Release (4.8.0)
Build id: 20180619-1200

When a try to install, the Eclipse throws this error:

Unable to read repository at https://updates.yakindu.com/solidity/releases/content.xml.
java.lang.RuntimeException: Unexpected error: java.security.InvalidAlgorithmParameterException: the trustAnchors parameter must be non-empty

**Comment by andreasmuelder** [comment]
This is fixed with Solidity Tools 0.7.3 release.
